docs/cli.md

```diff
@@ -320,6 +322,12 @@ and default benchmarking configuration:
 sb run --docker-image superbench/cuda:11.1 --host-file ./host.ini
 ```
 
+Run kernel launch benchmarks on host directly without using Docker:
+```bash title="SB CLI"
+sb run --no-docker --host-list localhost --config-override \
+  superbench.enable=kernel-launch superbench.env.SB_MICRO_PATH=/path/to/superbenchmark
```

Comment:
`SB_MICRO_PATH=/path/to/superbenchmark` =>
`SB_MICRO_PATH=/path/to/superbenchmark_micro_bin`?

Reply:
the env is for build prefix not bin directory?
https://github.com/microsoft/superbenchmark/blob/6d895da83c5b0490bf26d7c4cdf2903abbb660e3/superbench/benchmarks/micro_benchmarks/micro_base.py#L152-L154